{"name": "test_add[40-40-1]", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed!!!\nassert 1 == 80", "trace": "self = <Case.test_api.TestApi object at 0x101cc03a0>, a = 40, b = 40\nexpected = 1\n\n    @pytest.mark.parametrize(\"a,b,expected\", [(5, 7, 9),\n                                              (4, 4, 8),\n                                              (9, 9, 18),\n                                              (40, 40, 1),\n                                              (6, 5, 30)])\n    def test_add(self, a, b, expected):\n        sum = add(a, b)\n>       assert expected == sum, \"Test Failed!!!\"\nE       AssertionError: Test Failed!!!\nE       assert 1 == 80\n\n../PyAPI/Case/test_api.py:92: AssertionError"}, "attachments": [{"name": "log", "source": "94b84273-4914-4fbb-856f-fba57cf24780-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "3d3c764c-edef-4928-9c7e-7f1aa240a4c0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "40"}, {"name": "b", "value": "40"}, {"name": "expected", "value": "1"}], "start": 1585294252161, "stop": 1585294252161, "uuid": "1b86bfb0-7a2f-4e0b-b9c6-0eb71cc85d29", "historyId": "35d30ae0be8ffe96fc7e6e0d219fa78d", "testCaseId": "05c2483b3a18cb5ad38bc6850b3b725a", "fullName": "Case.test_api.TestApi#test_add", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "Parsons-Mac-mini.local"}, {"name": "thread", "value": "94963-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_api"}]}